js/test/calculator.test.js

```diff
@@ -0,0 +1,21 @@
+import calculator from '../calculator';
+
+test('It should perform addition, subtraction, multiplication, and division', () => {
+  expect(Object.keys(calculator)).toContain('add', 'subtract', 'multiply', 'divide');
+});
+
+test('It should add together any number of passed numbers', () => {
+  expect(calculator.add(1, 2, 3, 4, 5)).toEqual(15);
```

Comment:
- [ ] [OPTIONAL] It would be a good idea to have more than one test case. Try it with a different set of numbers and edge cases, like a parameter of the wrong type.